cmd/statusd/main.go

```diff
@@ -19,20 +19,34 @@ var (
-	httpPort       = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
-	ipcEnabled     = flag.Bool("ipc", false, "IPC RPC endpoint enabled")
-	cliEnabled     = flag.Bool("cli", false, "Enable debugging CLI server")
-	cliPort        = flag.String("cliport", debug.CLIPort, "CLI server's listening port")
-	logLevel       = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
-	logFile        = flag.String("logfile", "", "Path to the log file")
-	version        = flag.Bool("version", false, "Print version")
+	prodMode        = flag.Bool("production", false, "Whether production settings should be loaded")
+	nodeKeyFile     = flag.String("nodekey", "", "P2P node key file (private key)")
+	dataDir         = flag.String("datadir", params.DataDir, "Data directory for the databases and keystore")
+	networkID       = flag.Int("networkid", params.RopstenNetworkID, "Network identifier (integer, 1=Homestead, 3=Ropsten, 4=Rinkeby, 777=StatusChain)")
+	whisperEnabled  = flag.Bool("shh", false, "SHH protocol enabled")
+	swarmEnabled    = flag.Bool("swarm", false, "Swarm protocol enabled")
+	httpEnabled     = flag.Bool("http", false, "HTTP RPC endpoint enabled (default: false)")
+	httpPort        = flag.Int("httpport", params.HTTPPort, "HTTP RPC server's listening port")
+	ipcEnabled      = flag.Bool("ipc", false, "IPC RPC endpoint enabled")
+	cliEnabled      = flag.Bool("cli", false, "Enable debugging CLI server")
+	cliPort         = flag.String("cliport", debug.CLIPort, "CLI server's listening port")
+	logLevel        = flag.String("log", "INFO", `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`)
+	logFile         = flag.String("logfile", "", "Path to the log file")
+	version         = flag.Bool("version", false, "Print version")
+	printWhisperCfg = flag.Bool("printwhispcfg", false, "Print whisper config")
+	listenAddr      = flag.String("listenaddr", ":30303", "IP address and port of this node (e.g. 127.0.0.1:30303)")
+	identity        = flag.String("identity", "", "Protocol identity file (private key used for asymmetric encryption)")
+	passwordFile    = flag.String("passwordfile", "", "Password file (password is used for symmetric encryption)")
+	standalone      = flag.Bool("standalone", true, "Don't actively connect to peers, wait for incoming connections")
+	minPow          = flag.Float64("pow", params.WhisperMinimumPoW, "PoW for messages to be added to queue, in float format")
+	ttl             = flag.Int("ttl", params.WhisperTTL, "Time to live for messages, in seconds")
+
+	// MailServer
+	enableMailServer = flag.Bool("mailserver", false, "Delivers expired messages on demand")
+
+	// Push Notification
+	enablePN     = flag.Bool("notify", false, "Node is capable of sending Push Notifications")
+	firebaseAuth = flag.String("firebaseauth", "", "FCM Authorization Key used for sending Push Notifications")
```

Comment:
So here, when flags from both commands are merged, it becomes really hard to navigate them and make sense on what flag is related to what.

In fact, we have three on/off switches:
 - run LES protocol - on/off
 - run Whisper protocol - on/off
     - run Mailbox - on/off
(also -swarm, but it's mostly unused for now)
the rest of the flags are specific to these modes (les,whisper,whisper+mailbox).

It would be great to rename flags to facilitate this logical structure, for example rename `mailserver` to `shh.mailserver` (`shh` is a protocol name for Whisper). Most of the flags from `wnode-statusd` are actually related to `mailbox` and `shh`.

Reply:
following prefixes on flag name added 'shh.' 
     listenAddr   = flag.String("listenaddr", ":30303", "IP address and port of this node (e.g. 127.0.0.1:30303)")
    identity     = flag.String("shh.identity", "", "Protocol identity file (private key used for asymmetric encryptio    n)")
     passwordFile = flag.String("shh.passwordfile", "", "Password file (password is used for symmetric encryption)")
     standalone   = flag.Bool("standalone", true, "Don't actively connect to peers, wait for incoming connections")
     minPow       = flag.Float64("shh.pow", params.WhisperMinimumPoW, "PoW for messages to be added to queue, in float     format")
     ttl          = flag.Int("shh.ttl", params.WhisperTTL, "Time to live for messages, in seconds")
 
     // MailServer
     enableMailServer = flag.Bool("shh.mailserver", false, "Delivers expired messages on demand")
 
     // Push Notification
     enablePN     = flag.Bool("shh.notify", false, "Node is capable of sending Push Notifications")
     firebaseAuth = flag.String("shh.firebaseauth", "", "FCM Authorization Key used for sending Push Notifications")

is listenaddr whisper related or general option?
is standalone whisper related or general option?is listenaddr whisper related or general option?
is standalone whisper related or general option?